src/commons/api/history.rs

```diff
@@ -405,13 +328,40 @@ impl Default for CommandHistoryCriteria {
     }
 }
 
+//------------ CaCommandDetails ----------------------------------------------
+pub type CaCommandDetails = StoredCommand<CertAuth>;
+
+impl fmt::Display for CaCommandDetails {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        writeln!(f, "Time:   {}", self.time().to_rfc3339_opts(SecondsFormat::Secs, true))?;
+        writeln!(f, "Actor:  {}", self.actor())?;
+        writeln!(f, "Action: {}", self.details().summary().msg)?;
+
+        match self.effect() {
+            StoredEffect::Error { msg, .. } => writeln!(f, "Error:  {}", msg)?,
+            StoredEffect::Success { events } => {
+                writeln!(f, "Changes:")?;
+                for evt in events {
+                    writeln!(f, "  {}", evt)?;
```

Comment:
This used to write out `evt.details()`, is this a deliberate change?

Reply:
Yes. `evt` was a stored event containing meta information and the actual `details`. Now, the `evt` is just the details.